### DISCUSSION TITLE
PHP SDK: Call to undefined method FattureInCloud\Model\ModifySupplierRequest::setName()

### DISCUSSION THREAD

**Discussion by tobiascapin** [fattureincloud/api#140] [Q&A] [answered]
Buongiorno,
sto cercando di riprodurre l'esempio https://developers.fattureincloud.it/docs/sdks/php-sdk#-getting-started
```
$modify_supplier_request = new FattureInCloud\Model\ModifySupplierRequest; // \FattureInCloud\Model\ModifySupplierRequest | The modified Supplier.
$modify_supplier_request->setName("nuovo nome");
$modify_supplier_request->setPhone("03561234312");
```

Ma ottengo questo errore:
`Fatal error: Uncaught Error: Call to undefined method FattureInCloud\Model\ModifySupplierRequest::setName()`

Sto usando l'api da github caricata con composer 
`"fattureincloud/fattureincloud-php-sdk" : "~2.0"`

Ho lo stesso problema con ModifyClientRequest, avete qualche indicazione?

**Comment by emajo** [comment, marked as answer]
Ciao @tobiascapin, effettivamente c'è un errore, il codice dovrebbe essere:
```
$supplier = new FattureInCloud\Model\Supplier;
$supplier->setName("nuovo nome");
$supplier->setPhone("03561234312");

$modify_supplier_request = new FattureInCloud\Model\ModifySupplierRequest;
$modify_supplier_request->setData($supplier);
```

Grazie per la segnalazione, sistemeremo al più presto,
Emanuele

**Comment by tobiascapin** [comment]
Ok grazie mille. Infatti ero passato in json per bypassare il problema.
